app/tests/test_item_to_collection_propagation.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from datetime import datetime
+
+from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.geos import Polygon
+from django.test import TestCase
+
+from stac_api.models import Item
+from stac_api.utils import utc_aware
+
+import tests.database as db
+
+logger = logging.getLogger(__name__)
+
+
+class ItemsToCollectionTestCase(TestCase):
+    '''
+    Testing the propagation of item geometries to the bbox of the collection
+    '''
+    def setUp(self):
+        self.collection = db.create_collection('collection-1')
+        self.item = Item.objects.create(
+            collection=self.collection,
+            properties_datetime=utc_aware(datetime.utcnow()),
+            item_name='base-bbox',
+            geometry=GEOSGeometry(
+                'SRID=4326;POLYGON '
+                '((0 0, 0 45, 45 45, 45 0, 0 0))'
+            )
+        )
+        self.item.full_clean()
+        self.item.save()
+
+    def test_if_collection_gets_right_extent(self):
+        # the collection has to have the bbox of the item
+        self.assertEqual(self.collection.extent_geometry, self.item.geometry)
+
+    def test_if_new_collection_has_extent(self):
+        # a new collection as no bbox yet
+        collection_no_bbox = db.create_collection('collection-no-bbox')
+        self.assertIsNone(collection_no_bbox.extent_geometry)
+
+    def test_bigger_item(self):
```

Comment:
Maybe test name could be a bit more explicit: `test_changing_bbox_with_bigger_item`